packages/jest-console/src/BufferedConsole.ts

```diff
@@ -163,3 +167,9 @@ export default class BufferedConsole extends Console {
     return this._buffer.length ? this._buffer : undefined;
   }
 }
+
+function invariant(condition: unknown, message?: string): asserts condition {
```

Comment:
Should this be a `jest-util`? We also have one in `jest-circus`.

Reply:
`jest-util` doesn't work in the browser 😞 Ideally we'd just use `ts-invariant` or something, whenever https://github.com/apollographql/invariant-packages/pull/12 is merged

Reply:
Oh. I guess we need `@jest/invariant`. Kidding :smile: